### PR TITLE
Interface Improvements

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Improves the general interface to look less windows 2000 and adds some additional settings, while removing useless ones.

Wew

### CODE REVIEW

**Comment by BurgerLUA** [comment]
!review

File: interface/skin.dmf [hunk 1]
```diff
@@ -1423,168 +1011,69 @@ window "Telecomms IDE"
...
 		multi-line = true
-		is-password = false
 		no-command = true
 
 window "mainwindow"
 	elem "mainwindow"
 		type = MAIN
 		pos = 281,0
-		size = 640x440
+		size = 640x480
```
Comment (HeyBanditoz) [inline interface/skin.dmf]: ?????????¿¿¿¿¡

Comment (BurgerLUA) [reply interface/skin.dmf]: what the fuck does this even mean?

**Comment by skull132** [comment]
Neither the PR nor the changelog provide any insight into _what_ was actually changed. Please actually describe your modifications.

File: interface/skin.dmf (hunk 1 above)
Comment (HeyBanditoz) [reply interface/skin.dmf]: You tell me.

**Comment by BurgerLUA** [comment]
Changes:
- Removed Text Mode, which just made the screen black.
- Made the default map zoom 64x64.
- Added sprite scaling settings for the menus.
- Removed the borders for most of the elements.
- Made the horizontal and vertical slider togglable.
- Recolored the chat bar and buttons.
- Modified the font of the chat bar to be bigger.

**Comment by BurgerLUA** [comment]
The chat isn't bigger. That's just my personal setting after tinkering with the vertical scroll bar. When implemented, the split will be it's original 50/50.

**Comment by Lohikar** [comment]
Oh, I misread. Font size of the chat _bar_. Thought you were increasing the font size of chat in general.

**Pull request merged**